### ISSUE TITLE
[BUG] Failed to post event `state_update`

### ISSUE THREAD

**Issue by stwerner97** [voxel51/fiftyone#3166]
### System information

-   **OS Platform and Distribution**: ``Ubuntu 20.04``
-   **Python version**: ``Python 3.9.16``
-   **FiftyOne version**: ``FiftyOne v0.20.1, Voxel51, Inc.``
-   **FiftyOne installed from**: ``pip``

### Describe the problem

When I start the fiftyone app in a remote docker container, it throws a ``RuntimeError: Failed to post event `state_update` to http://0.0.0.0:<port>/event`` error when updating its view.

### Code to reproduce issue

````python
dataset_type = fo.types.COCODetectionDataset
  dataset = fo.Dataset.from_dir(
      dataset_dir=<path-to-mscoco>,
      dataset_type=dataset_type,
 )

view = dataset.take(100)

session = fo.launch_app(dataset, remote=True)    # I also tried remote=False
session.view = view    # this operation throws the error
````

### Other info/logs

````
Traceback (most recent call last):
<custom code traceback>
  File "/root/.cache/pypoetry/virtualenvs/mscoco-7Q30dl-p-py3.10/lib/python3.9/site-packages/fiftyone/core/session/session.py", line 220, in wrapper
    session._client.send_event(StateUpdate(state=session._state))
  File "/root/.cache/pypoetry/virtualenvs/mscoco-7Q30dl-p-py3.10/lib/python3.9/site-packages/fiftyone/core/session/client.py", line 129, in send_event
    self._post_event(event)
  File "/root/.cache/pypoetry/virtualenvs/mscoco-7Q30dl-p-py3.10/lib/python3.9/site-packages/fiftyone/core/session/client.py", line 177, in _post_event
    raise RuntimeError(
RuntimeError: Failed to post event `state_update` to http://0.0.0.0:5153/event
````

### What areas of FiftyOne does this bug affect?

-   [x] `App`: FiftyOne application issue
-   [ ] `Core`: Core Python library issue
-   [ ] `Server`: FiftyOne server issue

### Willingness to contribute

The FiftyOne Community encourages bug fix contributions. Would you or another member of your organization be willing to contribute a fix for this bug to the FiftyOne codebase?

-   [ ] Yes. I can contribute a fix for this bug independently
-   [ ] Yes. I would be willing to contribute a fix for this bug with guidance from the FiftyOne community
-   [x] No. I cannot contribute a bug fix at this time

**Comment by benjaminpkane** [comment]
Hi @stwerner97. Does upgrading to 0.21.0 solve this issue for you? I cannot reproduce with the `coco-2017` zoo dataset

**Comment by stwerner97** [comment]
Hi @benjaminpkane. No, unfortunately it does not solve the issue.

What can I do to help you reproduce the issue?

**Comment by benjaminpkane** [comment]
Pardon the delay. Starting the server directly with `python /path/to/fiftyone/server/main.py`, i.e.
```
python /root/.cache/pypoetry/virtualenvs/mscoco-7Q30dl-p-py3.10/lib/python3.9/site-packages/fiftyone/server/main.py --port 5153
```
will show the full stack trace from the server

**Comment by benjaminpkane** [comment]
This issue will be resolved in `v0.21.1`
